opta/core/aws.py

```diff
@@ -77,6 +78,23 @@ def delete_opta_config(self) -> None:
         if resp["ResponseMetadata"]["HTTPStatusCode"] != 204:
             raise Exception(f"Failed to delete opta config in {bucket}/{config_path}.")
 
+        s3_client = boto3.client("s3")
```

Comment:
(if i'm understanding the code structure properly) Deleting the s3 and dynamodb should be in `destroy_all()` and not in `delete_opta_config`

Reply:
I see where you were going, but actually, it should be on it's own function (where I just moved it)